### PR TITLE
[PAY-545] [PAY-542] Buy $AUDIO tile on $AUDIO page

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Changes the tile on the $AUDIO rewards page to have a "Buy with Coinbase" button and match the new redesign.

[Figma](https://www.figma.com/file/xSNINKrnGDcw1eKDap2zE5/%24AUDIO-Onramp-Flow?node-id=140%3A3268) 

Note:
- Does not have alternative payment button yet
- Does not have "A CREDIT CARD" copy since no alternative payment button yet
- Does not put the change of border radius behind feature flag
- "Buy with Coinbase" button does nothing

TODOs:
- [x] Put on $AUDIO & Rewards page behind feature flag
- [x] Get Coinbase logo for button

image

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

Will be updating this to the full changes once we have support for other payment providers.

Getting the Coinbase logo inline and sized appropriately was a bit trial-and-error heavy... it seems like we could do this with [some maths](https://iamvdo.me/en/blog/css-font-metrics-line-height-and-vertical-align) using a constant font ascender height but I was lazy and eyeballed it

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-cb-audio-page

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-cb-audio-page

File: packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        <Button
+          className={cn(styles.advancedButton, styles.receiveBtn)}
+          text={messages.receiveLabel}
+          textClassName={styles.textClassName}
+          onClick={onClickReceive}
+          leftIcon={<IconReceive className={styles.iconStyle} />}
+          type={ButtonType.GLASS}
+          minWidth={200}
+        />
+        <Button
```
Comment (dharit-tan) [inline packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx:114]: nit: This button looks a bit big on hover as well as the connect wallets button. Probably julian question. Also I realize the buttons will look different once there's also a stripe button, but the connect wallets button.

File: packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        />
+        {mobile && !IS_NATIVE_MOBILE && (
+          <MobileConnectWalletsDrawer onClose={onCloseConnectWalletsDrawer} />
+        )}
+      </div>
+    </div>
+  )
+}
+
+const ToggleCollapseButton = ({
```
Comment (dharit-tan) [inline packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx:136]: question for julian - do we care about animating this?

File: packages/web/src/pages/audio-rewards-page/Tiers.module.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
   background-color: var(--white);
   flex: 1;
   justify-content: flex-start;
-  border-radius: 6px;
+  border-radius: 12px;
```
Comment (dharit-tan) [inline packages/web/src/pages/audio-rewards-page/Tiers.module.css:8]: What are these for? Are they supposed to be part of this commit?

**Review by dharit-tan — APPROVED** [review]

File: packages/web/src/pages/audio-rewards-page/Tiers.module.css (hunk 3 above)
Comment (rickyrombo) [reply packages/web/src/pages/audio-rewards-page/Tiers.module.css:8]: yeah these border radiuses have been wrong for a while - confirmed with Julian/Forrest we want these to be 12px!

File: packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx:114]: Agreed - will want to do a follow up with design on this for sure.

File: packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx (hunk 2 above)
Comment (rickyrombo) [reply packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx:136]: will check with Julian in a design review: https://linear.app/audius/issue/PAY-551/design-review-for-dollaraudio-page-changes

**Pull request merged**